rescheduler/rescheduler.go

```diff
@@ -43,6 +49,9 @@ import (
 const (
 	criticalPodAnnotation      = "scheduler.alpha.kubernetes.io/critical-pod"
 	criticalAddonsOnlyTaintKey = "CriticalAddonsOnly"
+	// TaintsAnnotationKey represents the key of taints data (json serialized)
+	// in the Annotations of a Node.
+	TaintsAnnotationKey string = "scheduler.alpha.kubernetes.io/taints"
```

Comment:
It's needed for annotations cleanup in releaseAllTaintsDepracated function below.